experimental/generated/beats/fields.ecs.yml

```diff
@@ -3465,9 +3465,10 @@
     - name: agent.description
       level: extended
       type: keyword
-      ignore_above: 1024
       description: Free-form description of the GenAI agent provided by the application.
       example: Helps with math problems; Generates fiction stories
+      index: false
+      doc_values: false
```

Comment:
It's curious that doc_values is also false now but just setting `index: false` in the schema.

Is it possible to explicitly set `doc_values: true` in here to retain it in the mappings (and to keep `ignore_above`)?

https://github.com/elastic/ecs/blob/a1237a30337e855fe8dd97355783855e4e96044a/schemas/gen_ai.yml#L29-L37